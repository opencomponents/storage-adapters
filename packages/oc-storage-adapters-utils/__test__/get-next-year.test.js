const getNextYear = require('../get-next-year');

const DATE_TO_USE = new Date('2017');
const _Date = Date;
global.Date = jest.fn(() => DATE_TO_USE);
global.Date.UTC = _Date.UTC;
global.Date.parse = _Date.parse;
global.Date.now = _Date.now;

test('Get next year', () => {
  expect(getNextYear()).toMatchSnapshot();
});
