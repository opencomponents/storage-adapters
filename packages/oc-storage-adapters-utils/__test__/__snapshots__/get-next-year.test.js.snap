// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Get next year 1`] = `2018-01-01T00:00:00.000Z`;
